src/test/java/org/jabref/logic/integrity/DateContainedCheckerTest.java

```diff
@@ -0,0 +1,28 @@
+package org.jabref.logic.integrity;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class DateContainedCheckerTest {
+
+    private final DateContainedChecker checker = new DateContainedChecker();
+
+    @Test
+    void booktitleDoesNotAcceptIfDateIsContainedWithDash() {
+        assertNotEquals(Optional.empty(), checker.checkValue("European Conference on Circuit Theory and Design, {ECCTD} 2015, Trondheim, Norway, August 24-26, 2015"));
```

Comment:
Typically, we check `Optional.of(...)` -- could you change the tests to include the error message - or do you think, this is too much effort?